reactive/documentation/getting-started.adoc

```diff
@@ -0,0 +1,217 @@
+the Vert.x clients to connect to the databases of your choice:
+
+[source]
+[subs="verbatim,attributes"]
+.Gradle dependencies
+----
+implementation org.hibernate.reactive:hibernate-reactive-core:{hibernate-reactive-version}
+runtimeOnly io.vertx:vertx-pg-client:{vertx-version}
+runtimeOnly io.vertx:vertx-mysql-client:{vertx-version}
+runtimeOnly io.vertx:vertx-db2-client:{vertx-version}
+----
+
+== Configuration
+
+Hibernate Reactive is configured via the standard JPA `persistence.xml` 
+document which must be placed, as usual, in the `/META-INF` directory.
+
+The only configuration specific to Hibernate Reactive is the persistence
+`<provider>` element, which must be explicit:
+
+    <provider>org.hibernate.reactive.provider.ReactivePersistenceProvider</provider>
+
+Otherwise, configuration is almost completely transparent. Configure 
+Hibernate exactly as you usually would, noting that most configuration 
+properties related to JDBC or JTA aren't relevant in the context of 
+Hibernate Reactive.
+
+[source, XML]
+[subs="verbatim,attributes"]
+.Example for persistence.xml
+----
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd"
+             version="2.0">
+
+    <persistence-unit name="example">
+        <provider>org.hibernate.reactive.provider.ReactivePersistenceProvider</provider>
+
+        <class>org.hibernate.example.reactive.Author</class>
+        <class>org.hibernate.example.reactive.Book</class>
+
+        <properties>
+
+            <!-- PostgreSQL -->
+            <property name="javax.persistence.jdbc.url"
+	              value="jdbc:postgresql://localhost/hreact"/>
+
+            <!-- MySQL -->
+            <!--property name="javax.persistence.jdbc.url"
+	                 value="jdbc:mysql://localhost/hreact"/-->
+
+            <!-- Credentials -->
+            <property name="javax.persistence.jdbc.user" value="hreact"/>
+            <property name="javax.persistence.jdbc.password" value="hreact"/>
+
+            <!-- The Vert.x SQL Client connection pool size -->
+            <property name="hibernate.connection.pool_size" value="10"/>
+
+            <!-- Automatic schema export -->
+            <property name="javax.persistence.schema-generation.database.action"
+	              value="drop-and-create"/>
+
+            <!-- SQL statement logging -->
+            <property name="hibernate.format_sql" value="true"/>
+            <property name="hibernate.show_sql" value="true"/>
+
+        </properties>
+
+    </persistence-unit>
+
+</persistence>
+----
+
+== Obtaining a reactive session factory
+
+Obtain a Hibernate `SessionFactory` or JPA `EntityManagerFactory` 
+just as you normally would, for example, by calling:
+
+[source, JAVA]
+----
+    EntityManagerFactory emf = Persistence.createEntityManagerFactory("example");
```

Comment:
IMO it's odd to start via a "traditional" EntityManagerFactory as it seems to suggest that after this line I could use the traditional API, as a viable alternative. I think we need a custom boot API and amend such examples.

Reply:
I'm going to open an issue on the Reactive project. At the moment this is the way it works in reactive